### PR TITLE
Configure central-idp-issuer as env variable

### PR DESCRIPTION
**Pull request by ivan-degtiarenko**
## Description
Symmetric to https://github.com/stackrox/acs-fleet-manager/pull/382 changes, but with `central-idp-issuer` value. Do not merge until values are added to `app-interface`

## Checklist (Definition of Done)

- [ ] Unit and integration tests added
- [ ] Documentation added if necessary
- [ ] CI and all relevant tests are passing
- [ ] Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.

## Test manual

--

### CODE REVIEW

**Review by dhaus67 — APPROVED** [review]

File: templates/service-template.yml [hunk 1]
```diff
@@ -19,6 +19,10 @@ parameters:
   description: Which ACS Service Fleet Manager environment to use for this deployment
   value: production
 
+- name: CENTRAL_RHSSO_ISSUER
```
Comment (rukletsov) [inline templates/service-template.yml:22]:
```suggestion
- name: CENTRAL_IDP_ISSUER
```

File: Makefile [hunk 2]
```diff
@@ -700,11 +700,13 @@ deploy/service: OBSERVABILITY_CONFIG_TAG ?= "main"
 deploy/service: DATAPLANE_CLUSTER_SCALING_TYPE ?= "manual"
 deploy/service: CENTRAL_OPERATOR_OPERATOR_ADDON_ID ?= "managed-central-qe"
 deploy/service: FLEETSHARD_ADDON_ID ?= "fleetshard-operator-qe"
+deploy/service: CENTRAL_RHSSO_ISSUER ?= "https://sso.stage.redhat.com/auth/realms/redhat-external"
 deploy/service: deploy/envoy deploy/route
 	@if test -z "$(IMAGE_TAG)"; then echo "IMAGE_TAG was not specified"; exit 1; fi
 	@time timeout --foreground 3m bash -c "until oc get routes -n $(NAMESPACE) | grep -q fleet-manager; do echo 'waiting for fleet-manager route to be created'; sleep 1; done"
 	@oc process -f ./templates/service-template.yml \
 		-p ENVIRONMENT="$(FLEET_MANAGER_ENV)" \
+		-p CENTRAL_RHSSO_ISSUER="$(CENTRAL_RHSSO_ISSUER)" \
```
Comment (rukletsov) [inline Makefile:709]:
```suggestion
		-p CENTRAL_IDP_ISSUER="$(CENTRAL_IDP_ISSUER)" \
```

File: Makefile [hunk 3]
```diff
@@ -700,11 +700,13 @@ deploy/service: OBSERVABILITY_CONFIG_TAG ?= "main"
 deploy/service: DATAPLANE_CLUSTER_SCALING_TYPE ?= "manual"
 deploy/service: CENTRAL_OPERATOR_OPERATOR_ADDON_ID ?= "managed-central-qe"
 deploy/service: FLEETSHARD_ADDON_ID ?= "fleetshard-operator-qe"
+deploy/service: CENTRAL_RHSSO_ISSUER ?= "https://sso.stage.redhat.com/auth/realms/redhat-external"
```
Comment (rukletsov) [inline Makefile:703]:
```suggestion
deploy/service: CENTRAL_IDP_ISSUER ?= "https://sso.stage.redhat.com/auth/realms/redhat-external"
```

File: templates/service-template.yml [hunk 4]
```diff
@@ -19,6 +19,10 @@ parameters:
...
+  displayName: Central Red Hat SSO issuer
+  description: Which issuer to use to get tokens from Red Hat SSO in Central
```
Comment (rukletsov) [inline templates/service-template.yml:24]:
```suggestion
  description: OIDC issuer URL to pass to Central's auth config
```

File: templates/service-template.yml [hunk 5]
```diff
@@ -19,6 +19,10 @@ parameters:
   description: Which ACS Service Fleet Manager environment to use for this deployment
   value: production
 
+- name: CENTRAL_RHSSO_ISSUER
+  displayName: Central Red Hat SSO issuer
```
Comment (rukletsov) [inline templates/service-template.yml:23]:
```suggestion
  displayName: Central's IdP issuer
```

File: templates/service-template.yml [hunk 6]
```diff
@@ -1083,6 +1087,7 @@ objects:
             - --enable-db-debug=${ENABLE_DB_DEBUG}
             - --redhat-sso-client-id-file=/secrets/service/redhatsso-service.clientId
             - --redhat-sso-client-secret-file=/secrets/service/redhatsso-service.clientSecret
+            - --central-idp-issuer=${CENTRAL_RHSSO_ISSUER}
```
Comment (rukletsov) [inline templates/service-template.yml:1090]:
```suggestion
            - --central-idp-issuer=${CENTRAL_IDP_ISSUER}
```

**Review by rukletsov — COMMENTED** [review]
I believer you'll also need to update the AppSRE repo.

Any reason why you haven't included that into https://github.com/stackrox/acs-fleet-manager/pull/382?

**Review by dhaus67 — APPROVED** [review]

**Pull request merged**